### ISSUE TITLE
Source Map support

### ISSUE THREAD

**Issue by calebthebrewer** [calebthebrewer/gulp-selectors#15]
Adding support for a tool like `gulp-sourcemaps` would make it much easier to use this library in everyday pipelines. Not sure how it will work, but it is definitely worth looking into.

**Comment by toser** [comment]
Definitely! Would be great. Also for finding errors in live environment.

**Comment by kamlekar** [comment]
:+1:
